### PR TITLE
feat: add helm chart for querybook

### PR DESCRIPTION
**Pull request by yongchand**
Related to issue #562 
I know this is a huge commit so take some time to review.
It will definitely work when docker image "docker.pkg.github.com/pinterest/querybook/querybook" is substituted, but please double check since this is an official open source repo (I have checked in my own environment though).
There are several changes I made from k8s:

1. Add more components in labels just to be more official (Referenced from druid or kowl repo)
2. Add nodeselector, affinity, tolerations options for users
3. Use secret.yaml file to be closer to production level
4. Add enabled option for mysql, redis and elasticsearch

Please feel free to share opinions! Also please understand that this is a first draft!

PS. Special thanks to [Jaehyeok](https://github.com/JaeHyeokLee) from PUBG who helped me a lot for writing Querybook helm chart

### CODE REVIEW

**Comment by yongchand** [comment]
Also lint won't pass for helm template yaml
check: https://github.com/pre-commit/pre-commit-hooks/issues/420

**Comment by czgu** [comment]
> Also lint won't pass for helm template yaml
> check: [pre-commit/pre-commit-hooks#420](https://github.com/pre-commit/pre-commit-hooks/issues/420)

have you tried to add
```
- id: check-yaml
   exclude: ^helm/templates/
```
to .pre-commit-config.yaml?

**Comment by yongchand** [comment]
@czgu good point, thanks!

File: helm/values.yaml [hunk 1]
```diff
@@ -0,0 +1,140 @@
+worker:
+  replicaCount: 1
+  name: worker
+  image:
+    repository: docker.pkg.github.com/pinterest/querybook/querybook
```
Comment (czgu) [inline helm/values.yaml:5]: added official repo here https://hub.docker.com/r/querybook/querybook, can you please update

File: helm/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+appVersion: 2.4
+
+maintainers:
+  - name: Querybook
+    email: querybook@pinterest.com
+  - name: Yongchan Hong
+    email: yongchana@gmail.com
+  - name: Jaehyeok Lee
+    email: q32145@gmail.com
```
Comment (czgu) [inline helm/Chart.yaml:17]: minor: the linter should be fixed if you add a newline here

File: helm/values.yaml (hunk 1 above)
Comment (yongchand) [reply helm/values.yaml]: Yay, finally! Thank you so much :)

File: helm/Chart.yaml (hunk 2 above)
Comment (yongchand) [reply helm/Chart.yaml]: Oops, thank you 👍

**Review by czgu — APPROVED** [review]
Amazing work! thanks for getting it done so quickly 🎉

**Pull request merged**